recipes/atompaw/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "atompaw" %}
+{% set version = "4.1.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://users.wfu.edu/natalie/papers/pwpaw/{{ name }}-{{ version }}.tar.gz
+  sha256: 42a46c0569367c0b971fbc3dcaf5eaec7020bdff111022b6f320de9f11c41c2c
+  patches: 
+    - shared.patch
```

Comment:
You don't need to patch. You can do `./configure --enable-shared --disable-static`

Reply:
Ok, I did this